### PR TITLE
doc: nrf70: 54H: Fix typo

### PR DESCRIPTION
**Pull request by krish2718**
Fix typo due to copy-paste.

### CODE REVIEW

**Review by richabp — APPROVED** [review]

File: doc/nrf/device_guides/nrf70/nrf7002eb_54h20dk_gs.rst [hunk 1]
```diff
@@ -33,7 +33,7 @@ Pin mapping for nRF54H20 DK
 The pin mapping for the nRF54H20 DK is as follows:
 
 +-----------------------------------+-------------------+-----------------------------------------------+
-| nRF70 Series pin name (EB name)   | nRF54L15 PDK pins | Function                                      |
+| nRF70 Series pin name (EB name)   | nRF54h20 DK pins  | Function                                      |
```
Comment (richabp) [inline doc/nrf/device_guides/nrf70/nrf7002eb_54h20dk_gs.rst:36]:
```suggestion
| nRF70 Series pin name (EB name)   | nRF54H20 DK pins  | Function                                      |
```

**Review by richabp — CHANGES_REQUESTED** [review]
minor fix

**Review by kittydepa — APPROVED** [review]
Approving, but please see Richa's comment.

**Comment by krish2718** [comment]
> Approving, but please see Richa's comment.

Already fixed and pushed.

**Review by richabp — APPROVED** [review]

**Pull request merged**